source/npm/util.js

```diff
@@ -0,0 +1,29 @@
+const listrInput = require('listr-input');
+const chalk = require('chalk');
+const {throwError} = require('rxjs');
+const {catchError} = require('rxjs/operators');
+
+const handleNpmError = (error, task, message, executor) => {
```

Comment:
I extracted this out of the `publish-task-helper` as it's necessary for every npm command which uses OTP.